packages/react/src/components/LocaleModal/LocaleModalRegions.js

```diff
@@ -60,12 +60,28 @@ const LocaleModalRegions = ({
           .${prefix}--locale-modal__back .${prefix}--modal-close`
         );
 
+        /**
+         * Removes tabindex and role as it goes back
+         * @param {*} btn button to be handled
+         */
+        const localeBackPressed = btn => {
+          setIsFiltering(false);
+          setClearResults(true);
+          document.getElementById(`${prefix}--locale-modal__filter`).value = '';
+          btn.removeAttribute('tabindex');
+          btn.removeAttribute('role');
+        };
+
         [...localeBackBtn].forEach(btn => {
+          btn.setAttribute('tabindex', '1');
+          btn.setAttribute('role', 'button');
           btn.addEventListener('click', () => {
-            setIsFiltering(false);
-            setClearResults(true);
-            document.getElementById(`${prefix}--locale-modal__filter`).value =
-              '';
+            localeBackPressed(btn);
+          });
+          btn.addEventListener('keyup', e => {
```

Comment:
@RaphaelAmadeu do we need to remove the event listeners when `localeBackPressed` is called?

Reply:
I've implemented the removal of the listeners as it is activated.